src/engine/engineState.js

```diff
@@ -1019,6 +1029,7 @@ export class EngineState extends EventEmitter {
     }
     this.dirtyTxCache()
     this.onTxFetched(txid)
+    this.onTxidsChanged(this.getTxids())
```

Comment:
1. Why not call `this.onTxidsChanged(txid)`?
2. If the goal is to call this callback on every tx change, then adding it just here is not enough.

Reply:
`onTxidsChanged: (txids: Array<string>) => void` is the desired signature